pkg/development/devcontext.go

```diff
@@ -340,6 +341,19 @@ func distinguishGraphError(ctx context.Context, dispatchError error, source devi
 		}, nil
 	}
 
+	details, ok := spiceerrors.GetDetails[*errdetails.ErrorInfo](dispatchError)
```

Comment:
why doesn't the previous case handle this?

Reply:
Because the previous case is a non-gRPC wrapped error from the *local* dispatch vs a gRPC error from a remote dispatch